src/Db/Diff/PKSimDB_diff.txt

```diff
@@ -1,2 +1,67 @@
-UPDATE tab_calculation_method_rates SET formula='(Q*(1-HCT))*C_pls' WHERE calculation_method='MassTransferLink_forTwoPores' AND formula_rate='PARAM_MassTransferIntestineToMucosaPlasma';
-DELETE FROM tab_rate_generic_parameters WHERE calculation_method='MassTransferLink_forTwoPores' AND formula_rate='PARAM_MassTransferIntestineToMucosaPlasma' AND path_id=1524 AND parameter_name='Lymph flow rate';
+DELETE FROM tab_calculation_method_rates WHERE calculation_method='ActiveProcess_PKSim' AND formula_rate='RelExpBloodCellsNormGlobal';

```

Comment:
@Yuri05 
I removed all parameters, rate etc.. related to norm parameters
In other formula, removed the usage of norm parameters to use the non norm parameters automatically